### PR TITLE
update runtime and ci

### PR DESCRIPTION
**Pull request by ruslandoga**

### CODE REVIEW

File: .github/workflows/build.yml [hunk 1]
```diff
@@ -12,36 +12,36 @@ env:
 jobs:
   docker:
     name: docker build amd64
-    runs-on: ubuntu-latest
+    runs-on: ubuntu-22.04
```
Comment (ruslandoga) [inline .github/workflows/build.yml:15]: ensures same os version as in aws

File: .github/workflows/release.yml [hunk 2]
```diff
@@ -23,33 +23,13 @@ jobs:
       AWS_SECRET_ACCESS_KEY: ${{ secrets.AWS_SECRET_ACCESS_KEY }}
 
     steps:
-      - uses: actions/checkout@v2
-
-      # TODO use setup-beam once https://github.com/erlef/setup-beam/pull/124 is released
```
Comment (ruslandoga) [inline .github/workflows/release.yml:28]: it's released

File: mix.exs [hunk 3]
```diff
@@ -5,7 +5,7 @@ defmodule T.MixProject do
     [
       app: :t,
       version: "0.1.6",
-      elixir: "~> 1.13",
+      elixir: "~> 1.14",
```
Comment (ruslandoga) [inline mix.exs:8]: might be needed to use brew locally to update elixir to 1.14

```sh
> brew update && brew upgrade elixir erlang
```

File: .github/workflows/build.yml (hunk 1 above)
Comment (akhrail1996) [reply .github/workflows/build.yml:15]: will it become unsupported in 3 months or so?

Comment (ruslandoga) [reply .github/workflows/build.yml:15]: it's an LTS (long term support release) and it'd be supported for 6 years iirc

**Pull request merged**